### DISCUSSION TITLE
System Integrity Protection (SIP) on Orka Desktop macOS Virtual Machines

### DISCUSSION THREAD

**Discussion by tongnick** [macstadium/orka-desktop#10] [General]
Hi There, 

Occasionally need to use `dtruss`, however, it is pretty much knee-capped by SIP.

Is there a way to disable SIP on macOS VMs in Orka Desktop?

**Comment by celanthe** [comment]
Hi @tongnick ! Thank you for your question. We provide images with SIP disabled for testing purposes. You can install a selection of `no-sip` images from our new OCI-compliant image registry, [OrkaHub](https://orkahub.com/org/macstadium) or use the `no-sip` command to pull images locally as referenced in our documentation at: https://orkadocs.macstadium.com/docs/orka-engine-30-user-guide#pulling-remote-images-locally
